src/main/java/com/codessquad/qna/dto/PostDto.java

```diff
@@ -1,19 +1,16 @@
 package com.codessquad.qna.dto;
 
-import com.codessquad.qna.entity.Post;
-import com.codessquad.qna.util.DateFormat;
-
-import java.util.Date;
+import java.time.LocalDateTime;
 
 public class PostDto {
 
     private String title;
     private String author;
     private String body;
-    private String date;
+    private LocalDateTime date;
```

Comment:
```suggestion
    private LocalDateTime createdDateTime;
```

이렇게 바꿔주는 편이 훨씬 좋습니다.